packages/enhanced-img/package.json

```diff
@@ -44,10 +44,10 @@
 	},
 	"devDependencies": {
 		"@types/estree": "^1.0.5",
-		"@types/node": "^18.19.48",
+		"@types/node": "^20.17.16",
 		"rollup": "^4.27.4",
 		"svelte": "^5.2.9",
-		"typescript": "^5.6.3",
+		"typescript": "^5.3.3",
```

Comment:
This was changed to `^5.6.3` in https://github.com/sveltejs/kit/pull/12822/files#r1803038073 to support top-level jsdoc type imports. We can probably return it back to `^5.3.3` but replace the import statement with inline type imports https://github.com/sveltejs/kit/blob/09296d0f19c8d1ff57d699e637bd1beabb69d438/packages/enhanced-img/src/preprocessor.js#L1